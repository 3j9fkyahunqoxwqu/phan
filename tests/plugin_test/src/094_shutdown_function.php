<?php

register_shutdown_function('notafunction');  // Fails to emit PhanUndeclaredFunctionInCallable
register_shutdown_function(['MissingClass', 'notafunction']);  // Fails to emit PhanUndeclaredFunctionInCallable or anything else
