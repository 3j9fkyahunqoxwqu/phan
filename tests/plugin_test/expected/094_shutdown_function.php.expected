src/094_shutdown_function.php:3 PhanUndeclaredFunctionInCallable Call to undeclared function notafunction in callable
src/094_shutdown_function.php:4 PhanUndeclaredClassInCallable Reference to undeclared class \MissingClass in callable \MissingClass::notafunction
